src/llmq/quorums_instantsend.cpp

```diff
@@ -818,7 +818,14 @@ void CInstantSendManager::NotifyChainLock(const CBlockIndex* pindexChainLock)
 
 void CInstantSendManager::UpdatedBlockTip(const CBlockIndex* pindexNew)
 {
-    if (sporkManager.IsSporkActive(SPORK_19_CHAINLOCKS_ENABLED)) {
+    // TODO remove this after DIP8 has activated
+    bool fDIP0008Active;
+    {
+        LOCK(cs_main);
+        fDIP0008Active = VersionBitsState(chainActive.Tip()->pprev, Params().GetConsensus(), Consensus::DEPLOYMENT_DIP0008, versionbitscache) == THRESHOLD_ACTIVE;
```

Comment:
why not `pindexNew->pprev` (and no `cs_main`)?

Reply:
Oh...copy paste error...fixing this